servers/partner-ci/10.0/ui-fake.yaml

```diff
@@ -0,0 +1,234 @@
+---
+- parameter:
+    name: specific-parameters
+    parameters:
+      - text:
+          name: SPECIFIC_PROPS
+          default: |
+            UBUNTU_DIST=xenial;
+
+- project:
+    name: 'predefined_parameters'
+    fuel_release: '10.0'
+    fuel_release_checker: '10'
+    job_name: 'ui-fake'
+    nightly-name: '{fuel_release}.{job_name}.nightly'
+    custom_name: '{fuel_release}.{job_name}.custom'
+    reporter-name: '{fuel_release}.{job_name}.reporter'
+    nightly_timer: 'H 21 * * *'
+    nightly_disabled: false
+    force_erase: false
+    force_reuse: true
+    recreate_venv: false
+    debug: true
+    publish_results: true
+    fuel_ui_refs: ''
+    fuel_web_refs: ''
+    fuel_web_branch: 'origin/stable/newton'
+    fuel_ui_branch: 'origin/stable/newton'
+    docker_branch: 'fake-ui'
+    email_to: 'rkhozinov@mirantis.com'
+    scm_submodule_disabled: true
+    scm_wipe_workspace: false
+    skip_vote_successfull: false
+    skip_vote_failed: false
+
+    testrail_project: "Mirantis OpenStack"
+    testrail_suite: "[10.0][Fuel] UI testing"
+    testrail_suite_desc: "Suite description"
+    testrail_milestone: "10.0"
+    testrail_plan: "${{TREP_TESTRAIL_MILESTONE}} iso #${{SNAPSHOTS_ID}}"
+    testrail_plan_desc: "Test plan description"
+    testrail_run: "${{TREP_TESTRAIL_SUITE}}"
+    testrail_run_desc: "Test run description"
+    testrail_item_case: "${{TEST_GROUP}}"
+    trep_results_source: "xunit"
+    trep_debug_level: "WARNING"
+
+    testgroup_parameter_type: string
+    testgroup_value_type: default
+    testgroup_value: '{testgroup}'
+
+    testgroup:
+      - test_cluster_workflows
+      - test_deployment_history
+
+    jobs:
+      - '{nightly-name}':
+          multijob_build_pkg: false
+
+      - '{custom_name}.qa':
+          publish_results: false
+          scm_submodule_disabled: false
+
+      - '{custom_name}.dev':
+          scm_submodule_disabled: false
+          publish_results: false
+          testgroup_parameter_type: choice
+          testgroup_value_type: choices
+          testgroup_value: '{obj:testgroup}'
+
+      - '{fuel_release}.{job_name}.{testgroup}':
+          scm_submodule_disabled: false
+
+      - '{reporter-name}':
+          plugin_version: ''
+          plugin_name: {job_name}
+
+- runner-parameters: &runner-parameters
+    name: 'runner-parameters'
+    parameters:
+      - common-fuel-parameters:
+          debug:                 '{debug}'
+          snapshots_id:           ''
```

Comment:
Maybe snapshots_id, force_erase, force_reuse and fuel_release_checker are not needed